skimage/feature/corner.py

```diff
@@ -194,6 +312,17 @@ def hessian_matrix(image, sigma=1, mode='constant', cval=0, order='rc'):
     float_dtype = _supported_float_type(image.dtype)
     image = image.astype(float_dtype, copy=False)
 
+    if use_gaussian_derivatives is None:
+        use_gaussian_derivatives = False
+        warn("use_gaussian_derivatives currently defaults to False, but will "
```

Comment:
The tests fail here (see for example this [log](https://dev.azure.com/scikit-image/scikit-image/_build/results?buildId=7469&view=logs&j=eeafdf3e-6518-596e-3f31-4f6be6dd535c&t=42e5ba94-5a2b-5af3-282c-f8497361abb3&l=7566)) with the message
> NameError: name 'warn' is not defined

but warn is already imported... :weary:

Reply:
it was removed on `main`, so I think that was leading to an issue. I added it back after merging main here.